snapshots/overlay/overlay.go

```diff
@@ -22,25 +22,33 @@ package overlay
 import (
 	"context"
 	"fmt"
+	"io/ioutil"
```

Comment:
Go 1.16 release deprecated `io/ioutil` and all new code should use the preferred replacements.

Reply:
thanks :) updated